roboflow/core/version.py

```diff
@@ -446,6 +446,37 @@ def live_plot(epochs, mAP, loss, title=""):
 
             time.sleep(5)
 
+        if not self.model:
```

Comment:
This seems to be a copy-paste of [this code in Version.__init__](https://github.com/roboflow/roboflow-python/blob/main/roboflow/core/version.py#L106)

I'd include logic starting from [this line](https://github.com/roboflow/roboflow-python/blob/main/roboflow/core/version.py#L97), we would catch situation where model is not returned by API to be slightly more defensive

Reply:
added defensive check